src/docs/devices/kincony-kc868-a4s/index.md

```diff
@@ -0,0 +1,342 @@
+---
+title: KinCony KC868-A4S
+date-published: 2023-04-17
+type: relay
+standard: global
+---
+
+
+## GPIO Pinout
+
+| Pin    | Function                           |
+| ------ | ---------------------------------- |
+| GPIO36 | ANALOG_A1                          |
+| GPIO39 | ANALOG_A2                          |
+| GPIO34 | ANALOG_A3                          |
+| GPIO35 | ANALOG_A4                          |
+| GPIO4  | IIC_SDA                            |
+| GPIO16 | IIC_SCL                            |
+| GPIO5  | DS18B20/DHT11/DHT21/LED strip-1    |
+| GPIO14 | DS18B20/DHT11/DHT21/LED strip-2    |
+| GPIO23 | ETH_MDC                            |
+| GPIO18 | ETH_MDIO                           |
+| GPIO33 | RS485_RXD                          |
+| GPIO32 | RS485_TXD                          |
+| GPIO13 | GSM_RXD                            |
+| GPIO15 | GSM_TXD                            |
+
+[Additional pinout/design details](https://www.kincony.com/forum/forumdisplay.php?fid=41)
+
+## Basic Configuration
+
+```yaml
+esphome:
+  name: a4s
+  platform: ESP32
+  board: esp32dev
```

Comment:
You should be consistent with your device names, and also use the `esp32` block, not `platform: ESP32` inside `esphome` block.

```suggestion
esphome:
  name: KC868-A4S

esp32:
  board: esp32dev
  framework:
    type: arduino
  
```